base/shell/explorer/lang/en-US.rc

```diff
@@ -215,4 +215,5 @@ BEGIN
     IDS_RESTORE_ALL "&Show Open Windows"
     IDS_TRAYWND_UNDO_CASCADE "&Undo Cascade"
     IDS_TRAYWND_UNDO_TILE "&Undo Tile"
+    IDS_TRAYDESKBTN_TOOLTIP "Show Desktop/Windows"
```

Comment:
If I were you I would avoid the capitalization here. 

```suggestion
    IDS_TRAYDESKBTN_TOOLTIP "Show desktop/windows"
```
Cc @DarkFire01 @cbialorucki @Doug-Lyons regarding "good practices" in English here.

Reply:
Windows<7 and ReactOS uses uppercase but Windows7+ uses lowercase.

Win2k3:
![VirtualBox_Win2003_31_10_2024_00_06_01](https://github.com/user-attachments/assets/f8b0264d-8222-4fed-a4c6-b7c912ab9e37)

Win10:
![Win10TaskbarCM](https://github.com/user-attachments/assets/5eefafcd-225d-4993-900b-f2bb4f59a4bc)

Win2k3 (Quick Launch):
![VirtualBox_Win2003_31_10_2024_00_11_21](https://github.com/user-attachments/assets/40fbd68f-3f62-418a-a911-677805b7ccaa)